### DISCUSSION TITLE
petraptor does not attack enemy team

### DISCUSSION THREAD

**Discussion by girlyguppy** [FakeFishGames/Barotrauma#15025] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

in outpost deathmatch (could be other modes too), the petraptor does not attack the enemy team

### Reproduction steps

1. enter outpost deathmatch
2. hatch a petraptor egg
3. witness as it doesn't attack the enemies

### Bug prevalence

Just once

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in our private repo now, fix coming up in the next unstable build and the upcoming release.
